src/app/components/HorizontalRule/index.jsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import { C_POSTBOX, GEL_SPACING_DBL } from '../../lib/constants/styles';
+
+const HorizontalRule = styled.hr`
+  background-color: ${C_POSTBOX};
+  height: 0.25rem;
```

Comment:
I had questioned whether this could be GEL_SPACING_HLF but I don't feel strongly enough about it to hold up this work.